services/ir_pagerduty.rb

```diff
@@ -29,11 +29,11 @@ def attach_contact_methods_to_user(user)
   end
 
   def fetch_todays_schedules_users(sid)
-    today = Date.today
```

Comment:
This is much nicer, yes. Only tiny nitpick I would make is that there's now a possibility that `Date.today` could vary between calling `since_date` and `until_date` (very slim, obvs).